src/main/java/Duke/Storage.java

```diff
@@ -0,0 +1,113 @@
+package Duke;
+
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.time.LocalDate;
+import java.util.ArrayList;
+import java.util.Scanner;
+
+/**
+ * Handles the reading and writing of file
+ * which Duke accesses for the task list.
+ * Keeps the file path of the task list.
+ */
+
+public class Storage {
+    private String filePath;
+
+    /**
+     * A public constructor to create Storage.
+     * @param filePath The string for the file name.
+     */
+    public Storage(String filePath) {
+        this.filePath = filePath;
+    }
+
+
+//    public void createNew() throws IOException {
+//
+//        File f = new File(this.filePath);
+//        if (!f.exists()) {
+//            System.out.println("Creating new file... Please try again");
+//            f.createNewFile();
+//        }
+//    }
+
+    /**
+     * Loads and reads the content of the file to
+     * return an Array List of tasks saved in the file.
+     * @return An ArrayList of tasks.
+     * @throws IOException Thrown when the file is not found.
+     */
+    public ArrayList<Task> load() throws IOException {
+        ArrayList<Task> arr = new ArrayList<>();
```

Comment:
Rename the `arr` variable to something more descriptive? Like `fileContents` for example